res/values/strings.xml

```diff
@@ -311,9 +311,9 @@
     <string name="welcome_intro1_message">The messenger with the broadest audience in the world. Free and independent.</string>
     <string name="welcome_start_messaging">Start messaging</string>
     <string name="login_title">Log in</string>
-    <string name="login_header">Log in to your server</string>
-    <string name="login_explain">Log in with an existing email account</string>
-    <string name="login_subheader">For known email providers additional settings are set up automatically. Sometimes IMAP needs to be enabled in the web settings. Consult your email provider or friends for help.</string>
+    <string name="login_header">Log in to your email account/string>
```

Comment:
i think we really want to underline that is is the _server_ here. i would not change that.

Reply:
People that have e-mail and use it. People that know what server means. What has the broadest reach?

Reply:
it is not about  email vs. server.

it is more to make clear that they do not log in to _our_ server but to _their ones_.

there are apps out there that give the credentials - including plain text password - to some server driven by the app. this "your server" makes clearer that this is not the case here.